app/src/migrations/20200430000000-4-ipc-formversion-col.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'ipc_plan',
+          'formVersion',
+          {
+            allowNull: false,
+            comment: 'Version of form used to populate this plan',
+            type: Sequelize.STRING(30),
+            unique: false,
+            defaultValue: '1.0.0'
+          },
+          { transaction: t })
+      ]);
+    });
```

Comment:
I think you might be missing the step to backfill all existing records with it populated as 1.0.0 - not sure if the definition itself will do that for you.

Reply:
Migration did that on my local when I tested it
![image](https://user-images.githubusercontent.com/17445138/80766891-57a3d300-8afb-11ea-9bdd-1a992b1e8e7a.png)